kubelet/datadog_checks/kubelet/prometheus.py

```diff
@@ -553,6 +553,8 @@ def container_memory_cache(self, metric, scraper_config):
 
     def container_memory_rss(self, metric, scraper_config):
         metric_name = scraper_config['namespace'] + '.memory.rss'
+        # Filter out aberrant values
+        metric.samples = [sample for sample in metric.samples if sample[self.SAMPLE_VALUE] < 2**63]
```

Comment:
Please make the `**` computation a constant

Reply:
@ofek do you mean? something like

```suggestion
        max_value = 2**63
        metric.samples = [sample for sample in metric.samples if sample[self.SAMPLE_VALUE] < max_value]
```

Reply:
Yeah but somewhere it's run once, like the top of the file

Reply:
Something like this: https://github.com/DataDog/integrations-core/pull/13076/commits/f0ee37ca548d9dc80a81d8e2b69f92e4b5368131